internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/endToEnd/PartialUpdateTest.java

```diff
@@ -1430,16 +1436,17 @@ private void verifyConsumerThreadPoolFor(
             for (Map.Entry<String, Map<String, TopicPartitionIngestionInfo>> entry: topicPartitionIngestionContexts
                 .entrySet()) {
               Map<String, TopicPartitionIngestionInfo> topicPartitionIngestionInfoMap = entry.getValue();
-              Assert.assertTrue(topicPartitionIngestionInfoMap.size() <= 1);
               for (Map.Entry<String, TopicPartitionIngestionInfo> topicPartitionIngestionInfoEntry: topicPartitionIngestionInfoMap
                   .entrySet()) {
                 String topicPartitionStr = topicPartitionIngestionInfoEntry.getKey();
-                TopicPartitionIngestionInfo topicPartitionIngestionInfo = topicPartitionIngestionInfoEntry.getValue();
-                assertEquals(pubSubTopicPartition.toString(), topicPartitionStr);
-                assertTrue(topicPartitionIngestionInfo.getConsumerIdStr().contains(consumerPoolType.getStatSuffix()));
-                regionCount += 1;
+                if (pubSubTopicPartition.toString().equals(topicPartitionStr)) {
```

Comment:
This check won't work topicPartitionIngestionContexts map is empty. Should we do something like:
```java
assertFalse(topicPartitionIngestionContexts.isEmpty());
assertTrue(topicPartitionIngestionContexts.contains(pubSubTopicPartition.toString()));
```
?

Reply:
I think overall your concern is right. But this check will only work on the server host has topic partition we are interested in. As we loop through all server hosts, I would like to add counter logic for # of replicas found per region to ensure these checks passed with expected # for every region.

Reply:
>I would like to add counter logic for # of replicas found per region to ensure these checks passed with expected # for every region.

Yeah that sounds good. Thanks